spec/views/devise/registrations/verify_email.html.slim_spec.rb

```diff
@@ -0,0 +1,13 @@
+require 'rails_helper'
+
+describe 'devise/registrations/verify_email.html.slim' do
+  before do
+    allow(view).to receive(:email).and_return('foo@bar.com')
+  end
+
+  it 'contains link to resend confirmation page' do
+    render
+
+    expect(rendered).to have_link(t('links.resend'), href: new_user_confirmation_path)
+  end
```

Comment:
Add some more tests while you are there?  `has a localized heading` and `has a localized title` seem to be common ones.